en/subscription_services_tos.md

```diff
@@ -0,0 +1,111 @@
+# Mozilla Subscription Services | Terms of Service
+
+Version 1.0, effective October 11, 2022
+{: datetime="2022-10-11" }
+
+Mozilla Corporation offers various services to improve your privacy and security online, including Mozilla VPN and Firefox Relay (the “Services”).
+
+__Mozilla VPN__ is a Virtual Private Network in partnership with [Mullvad](https://mullvad.net). It keeps your internet traffic private from your internet service provider by encrypting and routing it through our partner’s networks instead. It will also replace your actual IP address to keep your personal IP address private from others, such as the websites you visit.
+
+__Firefox Relay__ allows you to keep your primary email address and phone number safe and private by making masks for your email address and phone number that allow you to send and receive emails, calls, and texts without revealing your actual contact information.
+
+Please read these Terms of Service carefully because they explain important information about your use of the Services.
+
+## Subscribing to Mozilla Services
+
+#### You’ll Need A Firefox Account
+A Firefox Account is required to use the Services. To create a Firefox Account, you will also need to agree to the [Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Privacy Notice](https://www.mozilla.org/privacy/firefox/) for your Firefox Account.
+
+#### Your Privacy
+The [Mozilla Subscription Services Privacy Notice](https://www.mozilla.org/privacy/subscription-services) explains what information is sent when you use the Services and how we handle and share that information.
+
+Mozilla is a global organization, and our computers and service providers’ computers may be in various countries around the world, including the United States. This means that your information might be processed on servers located outside of the country where you live, and that country may have a different level of data protection regulation than yours. By giving us information, you consent to this kind of transfer of your information.
+
+#### Your Payment
+__Payment.__ We offer the Services as automatically renewing subscription services. When you sign up, you authorize us to charge the payment method you provide for the subscription fees for the first term of the Service(s) you sign up for. Your plan renews automatically at the end of each term, and you authorize us to charge the subscription fees each term. If you stop paying, we will immediately suspend your account, and we may delete it after 2 months of not paying.
+
+__30-Day Refund Policy.__ The first time you subscribe to the Services through Mozilla's website, if you cancel your account within the first 30 days, you may request a refund and Mozilla will refund your first subscription term. This offer only applies the first time you subscribe. This is in addition to any rights you may have under your country’s consumer laws.
```

Comment:
Note: one new line doesn't do what you probably expect in markdown ([check here](https://github.com/mozilla/legal-docs/blob/2755a6632d1a670ee431cc419c2b4a2dd7fb298a/en/subscription_services_tos.md)). I think the result will be the same when converted to HTML

Reply:
I don't think I understand. But the preview looks correct.